src/deno/_pointers.ts

```diff
@@ -32,6 +46,37 @@ export function denoFromPlatformPointer<T>(value: PlatformPointer<T>): Pointer<T
   return new Pointer(value as unknown as PlatformPointer<T>);
 }
 
+export function denoToPlatformStruct(data: TypedArray): Uint8Array;
+export function denoToPlatformStruct<T extends AllocatableStruct>(
+  data: Pointer<T>,
+  dataType: AllocatableStructConstructor<T>,
+): Uint8Array;
+export function denoToPlatformStruct<T extends AllocatableStruct>(
+  data: TypedArray | Pointer<T>,
+  dataType: AllocatableStructConstructor<T>,
+): Uint8Array;
```

Comment:
These overloads don't actually do anything and can never be called anyway because all these functions are called through the `Platform` interface.